TKeazirian.HTTPServer/Handler/HtmlHandler.cs

```diff
@@ -2,17 +2,20 @@ namespace TKeazirian.HTTPServer.Handler;
 
 using Request;
 using Response;
+using Helpers;
 
 public class HtmlHandler : Handler
 {
     public override Response HandleResponse(Request request)
     {
         string body = "<html><body><p>HTML Response</p></body></html>";
+        byte[] bodyBytes = ByteConverter.ToByteArray(body);
```

Comment:
html = HyperTEXT Markup Language.  Do we really want to convert it to bytes here?

Reply:
Well, the entire response object was converted to `byte[]`  before sending it to the server.  So I (believe if I'm understanding correctly) that it's just doing this conversion step sooner.  

This is what it was doing originally in `Server.cs`:

```csharp
Response response = _router.Route(request);
byte[] encodedResponse = Encoding.ASCII.GetBytes(response.FormatResponse());
 socket.Send(encodedResponse, SocketFlags.None);
```

Reply:
OK, I can buy that.  Reference my [other comment](https://github.com/t-keazirian/HTTPServer/pull/36#discussion_r932346391) about the renaming (which I did not see as necessary)

Reply:
also, if you are going to do this encoding for every and all things, is there a way to do it without so much code duplication? (could you just do the encoding in the Response Builder's `addBody` method for instance?

Reply:
Agreed about the renaming - I'll fix!  Thanks!

Reply:
> also, if you are going to do this encoding for every and all things, is there a way to do it without so much code duplication? (could you just do the encoding in the Response Builder's `addBody` method for instance?

Oh interesting idea...I hadn't thought of that!  Let me try that out...

Reply:
It doesn't appear that it will work this way for images.  If I convert text/string body to bytes in `SetBody`, no problem but if I send in the image from the file as a string to `SetBody` and then convert it to the Bytes there, it errors.